ambari-project/pom.xml

```diff
@@ -344,7 +345,7 @@
       <dependency>
         <groupId>commons-codec</groupId>
         <artifactId>commons-codec</artifactId>
-        <version>1.8</version>
+        <version>${commons.codec.version}</version>
```

Comment:
Since we use the dependencyManagement section at ambari-project/pom.xml , how about inheriting dependency version at all child projects ?
https://stackoverflow.com/a/38882340